### DISCUSSION TITLE
Is it possible to add gRPC response code to default metric

### DISCUSSION THREAD

**Discussion by IgorMamushin** [open-telemetry/opentelemetry-dotnet#5863] [Q&A]
I have gRPC service. It has only gRPC endpoints. And all the time when i get 200 http code even if I send RpcException with internal status code.

Is it possible to add rpc_response_status_code next to http_response_status_code in http_server_request_duration_seconds asp metric?

**Comment by cijothomas** [comment]
This is best discussed in the contrib repo which hosts grpc instrumentation. Any changes to attributes must come via semantic conventions, so need to open an issue (if not existing) to semantic conventions repo.
